lib/application/i18n/assets/en.json

```diff
@@ -3162,10 +3165,18 @@
   "navajo_source_dictionary": "whole Dictionary",
   "navajo_source_alphabet": "only Alphabet",
   "nonogramsolver_title": "Nonogram Solver",
-  "nonogramsolver_description": "Creates a solution for a given Nonogram",
+  "nonogramsolver_description": "Creates a solution for a given Nonogram (Japanese crosswords)",
   "nonogramsolver_dataerror": "Error in the data",
   "nonogramsolver_hinterror": "Error in the hint data",
   "nonogramsolver_createhint": "It cannot be guaranteed that the puzzle is unique. Check beforehand.",
+  "nonogramsolver_setup_generator": "Setup Generator",
+  "nonogramsolver_setup_solver": "Setup Solver",
+  "nonogramsolver_generator_loadfrom_hint": "Generate Nonogram from image, load structure from JSON/TXT file (Nonogram.org or GC Wizard formatted) or draw manually.",
+  "nonogramsolver_solver_loadfrom_hint": "Load from JSON/Text file (Nonogram.org or GCWizard formatted) or enter size and hints manually:",
+  "nonogramsolver_loadfromimage": "Load from Image",
```

Comment:
Also da bin ich nicht so ganz einverstanden. Es wird ja kein Bild von einem Nonogram importiert, sondern von einem Bild ein Nonogram erstellt. Finde ich nicht richit, Eher "Import a Image"

Reply:
Yes, thats why the hint text clarifies it above the button ;)